tests/net/hostname/src/main.c

```diff
@@ -344,4 +385,22 @@ ZTEST(net_hostname, test_hostname_set)
 	}
 }
 
+ZTEST(net_hostname, test_hostname_event)
```

Comment:
Personally, I'd just compile out the entire test case in case mgmt is disabled, instead of leaving an empty one to execute.

Reply:
Done